core/kits/manager.php

```diff
@@ -287,12 +287,16 @@ private function before_delete_kit( $post_id, $is_permanently_delete = false ) {
 	 * @return array $admin_bar_config
 	 */
 	private function add_menu_in_admin_bar( $admin_bar_config ) {
-		$admin_bar_config['elementor_edit_page']['children'][] = [
-			'id' => 'elementor_site_settings',
-			'title' => __( 'Site Settings', 'elementor' ),
-			'sub_title' => __( 'Site', 'elementor' ),
-			'href' => $admin_bar_config['elementor_edit_page']['href'] . '#' . self::E_HASH_COMMAND_OPEN_SITE_SETTINGS,
-		];
+		$document = Plugin::$instance->documents->get( get_the_ID() );
+
+		if ( $document ) {
```

Comment:
Check is built with Elementor. if not - query the last page edited with Elementor.

Reply:
Handled.